org.eclipse.scanning.points/src/org/eclipse/scanning/points/validation/ScanRequestValidator.java

```diff
@@ -55,11 +55,11 @@ public void validate(ScanRequest<?> req) throws ValidationException, Instantiati
 		try {
 			Map<String, Object> dmodels = req.getDetectors();
 			if (dmodels!=null) {
-				if (dmodels.isEmpty()) throw new ModelValidationException("The detector models are empty!", req, "detectors");
-			    validateMalcolmRules(dmodels);
-		        validateDetectors(dmodels);
-		        validateAnnotations(dmodels);
-
+				if (!dmodels.isEmpty()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1066)